### PR TITLE
refactor: move to react wrapper for slides

### PR DESCRIPTION
**Pull request by DSchau**
Note: the webpack config is currently tied to a commit hash; we'll want to fix that once this (objectpartners/object-partners-presentation-config#6) is merged

### CODE REVIEW

**Comment by jeffsheets** [comment]
I tried the branch locally and got this error, but i'm logging off for the weekend and I'll dig into it some more on Monday
```ERROR in multi (webpack)-dev-server/client?http://localhost:9998 ./src/main
Module not found: Error: Can't resolve './src/main' in '/Users/jeffsheets/projects/react-redux-workshop/slidedeck'
 @ multi (webpack)-dev-server/client?http://localhost:9998 ./src/main
```

**Comment by DSchau** [comment]
Did you use yarn? Yarn seems to over aggressively cache git repos unfortunately. I'll have to get that PR approved and tagged before we can merge this

**Comment by jeffsheets** [comment]
Oh yep. `yarn cache clean` then removing node_modules and doing a new `yarn` fixed it. Thanks for the tip.

**Comment by DSchau** [comment]
@jeffsheets wanna give this one a shot?

File: slidedeck/src/components/slide-deck/slide-deck.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import * as renderHtml from 'react-render-html';
+
+import 'reveal.js/css/reveal.css';
+import 'object-partners-revealjs-theme';
+import 'highlight.js/styles/monokai.css';
+
+import '!style-loader!css-loader?importLoaders=1!postcss-loader?config=node_modules/object-partners-presentation/dist/config-files/postcss.config.js!sass-loader!./slide-deck.scss'; // hack hack hack
```
Comment (DSchau) [inline slidedeck/src/components/slide-deck/slide-deck.tsx]: Note my comment in 4efdc19

**Comment by DSchau** [comment]
niceeeee

**Pull request merged**